inc/API.php

```diff
@@ -94,8 +99,22 @@ public static function peerboard_api_call($slug, $token = 0, $body, $type = 'GET
 
     if ($check_success) {
       $success = self::check_request_success($request, func_get_args());
-
-      if (!$success) {
+      // If we have issue with ssl
+      if ($success === 'ssl_fix') {
+        // Try to update certificate and fix ssl issue 
+        $ssl_fix = self::update_wp_ca_bundle();
+
+        if (isset($ssl_fix['success'])) {
+          // Make the same request
+          $success = self::make_api_req_again($slug, $token, $body, $type, $api_url);
+          // the last solution disable ssl
+          if (!$success) {
+            self::make_api_req_again($slug, $token, $body, $type, $api_url, $check_success = true, $ssl_verify = false);
```

Comment:
Looks like a return or request update is missing here. It's not clear why the extra "make_api_req_again" function if it just changes two args.

Reply:
I removed, additional function, it was needed for other logic but then I changed it and forgot to remove